### ISSUE TITLE
move from readthedocs to alternate

### ISSUE THREAD

**Issue by youngblood** [ContinuumIO/topik#43]
perhaps GitHub Pages?

**Comment by ahmadia** [comment]
Any particular reason we want to stop using readthedocs?  It's pretty much the standard for freely posting documentation and not worrying about rebuilding.

**Comment by msarahan** [comment]
Yes, we want to be able to generate plots in the docs using Bokeh, and ReadTheDocs will not allow Sphinx extensions like this.  Basically, I want to make sure that the docs are generated and run their contained examples, and when they fail to generate, it means that the docs are out of date.

We will move to something that is no less automated than ReadTheDocs - Travis building the docs, then pushing to github pages.

**Comment by ahmadia** [comment]
Yup, sounds good.  Requires a little bit of trickery in saving credentials/encryption within Travis.
